src/com/jetbrains/Duke.java

```diff
@@ -21,38 +20,50 @@ public static void main(String[] args) {
         while (!input.equals("bye")) {
             Task task;
             try {
-            if (input.equals("list")) {
-                displayList(list);
-                System.out.println("\n");
-            } else if (input.contains("done")) {
-                String[] doneCommand = input.split(" ");
-                task = list.get(Integer.parseInt(doneCommand[1]) - 1);
-                System.out.println("Good job! I've marked this task as done:\n    " +
-                        task.markDone() +
-                        "\n");
-            } else if (input.contains("todo") ||
-                    input.contains("deadline") ||
-                    input.contains("event")) {
+                if (input.equals("list")) {
+                    displayList(list);
+                    System.out.println("\n");
+                } else if (input.contains("done")) {
+                    String[] command = input.split(" ");
+                    task = list.get(Integer.parseInt(command[1]) - 1);
+                    System.out.println("Good job! I've marked this task as done:\n    " +
+                            task.markDone() +
+                            "\n");
+                } else if (input.contains("delete")) {
```

Comment:
do you think maybe switch statements could be used to make the code look more concise?